### PR TITLE
Sends user verification emails to account when creating account not on dev server

### PR DESCRIPTION
**Pull request by nlumiere**

### CODE REVIEW

File: proj/src/main/webapp/chat.js [hunk 1]
```diff
@@ -318,18 +318,33 @@ const LIMIT = 20; // how many messages to load at a time
...
-                initRef();
-                populateProfileSidebar(firebaseUser.uid);
-                addBlockedToSettings();
-            });
+            // If the user is verified or using a test server
+            if(firebaseUser.emailVerified || window.location.href.includes("https://8080")){
+                if(firebaseUser.emailVerified)
+                    console.log("You have been verified as a Camaraderie user");
+                else
+                    console.log("You have been verified as a Camaraderie testing developer");
```
Comment (aokeke) [inline proj/src/main/webapp/chat.js]: Can you put curly braces around the parts in the if/else. It probably doesn't need it, but just for my sanity.

File: proj/src/main/webapp/chat.js [hunk 2]
```diff
@@ -318,18 +318,33 @@ const LIMIT = 20; // how many messages to load at a time
...
-            // InitUserChat sets information relevant to logged-in user
-            // Must run before enclosed functions
-            await initUserChat().then(function(){
-                populateSidebar();
-                initRef();
-                populateProfileSidebar(firebaseUser.uid);
-                addBlockedToSettings();
-            });
+            // If the user is verified or using a test server
+            if(firebaseUser.emailVerified || window.location.href.includes("https://8080")){
```
Comment (aokeke) [inline proj/src/main/webapp/chat.js:322]: It seems weird to have something for the devs in the deployed code. Is there any other way to do this? Like maybe we can check if the user has a specific exemption property in the database that the devs can manually add from the firebase database page?

**Review by aokeke — APPROVED** [review]
LGTM but a couple comments.

**Review by leeannachen — APPROVED** [review]
lgtm

File: proj/src/main/webapp/chat.js (hunk 2 above)
Comment (nlumiere) [reply proj/src/main/webapp/chat.js:322]: Ideally we would be able to create admin accounts and whatnot, but given the very constrained time at this point I don't think it makes sense to build an admin/dev account feature.

**Pull request merged**